Detectors/CTP/workflow/src/ctp-reco-workflow.cxx

```diff
@@ -0,0 +1,63 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// @file   ctp-reco-workflow.cxx
+/// @author RL from CPV example
+/// @brief  Basic DPL workflow for CTP reconstruction starting from digits
+#include "Framework/WorkflowSpec.h"
+#include "Framework/ConfigParamSpec.h"
+#include "CTPWorkflow/RecoWorkflow.h"
+#include "Algorithm/RangeTokenizer.h"
+#include "CommonUtils/ConfigurableParam.h"
```

Comment:
In order to create a correct DPL `DataHeadr.firstTForbit` in the root file digits driven mode , the workflows should have the lines like https://github.com/AliceO2Group/AliceO2/blob/59df0714e1705fb6e3d55b8f0bda869f2a2b866f/Detectors/ITSMFT/ITS/workflow/src/its-reco-workflow.cxx#L15 
https://github.com/AliceO2Group/AliceO2/blob/59df0714e1705fb6e3d55b8f0bda869f2a2b866f/Detectors/ITSMFT/ITS/workflow/src/its-reco-workflow.cxx#L40
https://github.com/AliceO2Group/AliceO2/blob/59df0714e1705fb6e3d55b8f0bda869f2a2b866f/Detectors/ITSMFT/ITS/workflow/src/its-reco-workflow.cxx#L72-L73

So, add here `#include "DetectorsRaw/HBFUtilsInitializer.h"`

Reply:
I added include and also equivalents of lines 40 and 73 ?

Reply:
ok, your next two comments resolving this.